src/com/namelessmc/java_api/NamelessUser.java

```diff
@@ -1,54 +1,45 @@
 package com.namelessmc.java_api;
 
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Optional;
-import java.util.UUID;
-import java.util.stream.Collectors;
-import java.util.stream.StreamSupport;
-
-import org.apache.commons.lang3.Validate;
-
 import com.google.gson.Gson;
 import com.google.gson.JsonArray;
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 import com.namelessmc.java_api.Notification.NotificationType;
 import com.namelessmc.java_api.RequestHandler.Action;
-import com.namelessmc.java_api.exception.AccountAlreadyActivatedException;
-import com.namelessmc.java_api.exception.AlreadyHasOpenReportException;
-import com.namelessmc.java_api.exception.InvalidValidateCodeException;
-import com.namelessmc.java_api.exception.ReportUserBannedException;
-import com.namelessmc.java_api.exception.UnableToCreateReportException;
+import com.namelessmc.java_api.exception.*;
+import org.apache.commons.lang3.Validate;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
 
 public final class NamelessUser {
 
 	private final NamelessAPI api;
 	private final RequestHandler requests;
-	
+
 	private Integer id;
 	private String username;
 	private Optional<UUID> uuid;
 	private Optional<Long> discordId;
 
 	private JsonObject userInfo;
-	
+
 	// only one of id, username, uuid, discordId has to be provided
-	NamelessUser(final NamelessAPI api, final Integer id, final String username, final Optional<UUID> uuid, final Long discordId) throws NamelessException {
+	NamelessUser(final NamelessAPI api, final Integer id, final String username, final Optional<UUID> uuid, final Long discordId) {
```

Comment:
Since you changed arguments from `null` to `-1` you can change object types to primitives here